tests/test-helpers.js

```diff
@@ -0,0 +1,31 @@
+const { Writable } = require("stream")
+
+async function runMiddleware(fn, req) {
```

Comment:
Can we inline this helper into the test file? Since it's just 1 and pretty small I don't see a problem having it here

Reply:
Sure thing! Inlined it to the same file :+1: